commit bc3f15a08bbb100055278f7a2fa3355c8d4c05f1
Author: Mark Robert Miller <markrmiller@apache.org>
Date:   Sat Oct 27 16:26:36 2012 +0000

    raise wait from 30 to 45 - my freebsd vm still trips 30 sometimes
    
    git-svn-id: https://svn.apache.org/repos/asf/lucene/dev/trunk@1402818 13f79535-47bb-0310-9956-ffa450edef68

diff --git a/solr/core/src/test/org/apache/solr/cloud/SyncSliceTest.java b/solr/core/src/test/org/apache/solr/cloud/SyncSliceTest.java
index 9f3a00b..5d2a0dd 100644
--- a/solr/core/src/test/org/apache/solr/cloud/SyncSliceTest.java
+++ b/solr/core/src/test/org/apache/solr/cloud/SyncSliceTest.java
@@ -162,7 +162,7 @@ public class SyncSliceTest extends AbstractFullDistribZkTestBase {
     // to talk to a downed node causes grief
     waitToSeeDownInClusterState(leaderJetty, jetties);
 
-    waitForThingsToLevelOut(30);
+    waitForThingsToLevelOut(45);
     
     checkShardConsistency(false, true);
     
